components/org.wso2.carbon.identity.api.server.claim.management/org.wso2.carbon.identity.rest.api.server.claim.management.v1/src/gen/java/org/wso2/carbon/identity/rest/api/server/claim/management/v1/ClaimManagementApi.java

```diff
@@ -368,6 +419,31 @@ public Response updateClaimDialect(@ApiParam(value = "Id of the claim dialect.",
         return delegate.updateClaimDialect(dialectId,claimDialect);
     }
 
+    @Valid
+    @PUT
+    @Path("/import")
+    @Consumes({ "multipart/form-data" })
+    @Produces({ "application/json" })
+    @io.swagger.annotations.ApiOperation(value = "Update claim dialect with related claims from XML, YAML, or JSON file",
+            notes = "This API provides the capability to update a claim dialect and all related claims from a file in XML, YAML, or JSON format. <br><b>Permission required:</b> <br> * /permission/admin/manage/identity/claimmgt/metadata/create <br> <b>Scope required:</b> <br> * internal_claim_meta_create",
+            response = void.class)
+    @io.swagger.annotations.ApiResponses(value = { 
+        @io.swagger.annotations.ApiResponse(code = 201, message = "Item Created."),
+        
+        @io.swagger.annotations.ApiResponse(code = 400, message = "Invalid input request."),
+        
+        @io.swagger.annotations.ApiResponse(code = 401, message = "Unauthorized."),
+        
+        @io.swagger.annotations.ApiResponse(code = 403, message = "Resource Forbidden."),
+        
+        @io.swagger.annotations.ApiResponse(code = 500, message = "Internal Server Error.") })
+
+    public Response updateClaimDialectFromFile(@ApiParam(value = "The file to be uploaded.") @Multipart(value = "file") InputStream fileInputStream,
+    @ApiParam(value = "The file to be uploaded. : details") @Multipart(value = "file" ) Attachment fileDetail) {
+
+        return delegate.updateClaimDialectFromFile(fileInputStream,fileDetail);
```

Comment:
Add a query parameter for adding preference to delete existing claims when updating